.circleci/ds054_fasttrack_outputs.txt

```diff
@@ -9,12 +9,16 @@ fmriprep/logs/CITATION.tex
 fmriprep/sub-100185
 fmriprep/sub-100185/func
 fmriprep/sub-100185/func/sub-100185_task-machinegame_run-1_boldref.nii.gz
+fmriprep/sub-100185/func/sub-100185_task-machinegame_run-1_desc-acompcor_mask.nii.gz
 fmriprep/sub-100185/func/sub-100185_task-machinegame_run-1_desc-brain_mask.json
 fmriprep/sub-100185/func/sub-100185_task-machinegame_run-1_desc-brain_mask.nii.gz
 fmriprep/sub-100185/func/sub-100185_task-machinegame_run-1_desc-confounds_timeseries.json
 fmriprep/sub-100185/func/sub-100185_task-machinegame_run-1_desc-confounds_timeseries.tsv
+fmriprep/sub-100185/func/sub-100185_task-machinegame_run-1_desc-CSF_mask.nii.gz
 fmriprep/sub-100185/func/sub-100185_task-machinegame_run-1_desc-preproc_bold.json
 fmriprep/sub-100185/func/sub-100185_task-machinegame_run-1_desc-preproc_bold.nii.gz
+fmriprep/sub-100185/func/sub-100185_task-machinegame_run-1_desc-tcompcor_mask.nii.gz
```

Comment:
```suggestion
fmriprep/sub-100185/func/sub-100185_task-machinegame_run-1_desc-CompCorTemp_mask.nii.gz
```

a/tCompCor can be determined from the ROIs used. If we use `desc-CompCorXXX` then all the compcor-related masks will go together when files are listed ordered by name, which is convenient.